demisto_sdk/commands/doc_reviewer/tests/doc_reviewer_test.py

```diff
@@ -311,6 +315,166 @@ def test_failure_on_malformed_rns(self, pack):
         assert not result
 
 
+class TestDocReviewXSOAROnly:
+
+    """
+    Tests for the `--xsoar-only` flag.
+    """
+
+    default_args = ["--xsoar-only"]
+
+    class CommandResultCode(Enum):
+
+        """
+        Holds result code for the execution of `doc-review` command.
+        """
+
+        SUCCESS = 0
+        FAIL = 1
+
+    def run_doc_review_cmd(self, cmd_args: List[str]) -> Result:
+
+        """
+        Uses the Click CLI runner to invoke a command with input arguments and returns the result
+        """
+
+        args: List[str] = self.default_args + cmd_args
+
+        return CliRunner().invoke(__main__.doc_review, args)
+
+    def test_valid_supported_pack(self, supported_pack: Pack):
+
+        """
+        Given -
+            An XSOAR-supported Pack with correct spelling.
+
+        When -
+            Running `doc-review` on XSOAR-supported Pack with `--xsoar-only` flag set.
+
+        Then -
+            Ensure `doc-review` succeeds.
+        """
+
+        cmd_args: List[str] = [
+            "--input",
+            supported_pack.path,
+        ]
+
+        result = self.run_doc_review_cmd(cmd_args)
+
+        assert result.exit_code == self.CommandResultCode.SUCCESS.value
+
+    def test_valid_non_supported_pack(self, non_supported_pack: Pack):
+
+        """
+        Given -
+            A non-XSOAR-supported Pack with correct spelling.
+
+        When -
+            Running `doc-review` on a non-XSOAR-supported Pack with `--xsoar-only` flag set.
+
+        Then -
+            Ensure `doc-review` succeeds.
+        """
+
+        cmd_args: List[str] = [
+            "--input",
+            non_supported_pack.path,
+        ]
+
+        result = self.run_doc_review_cmd(cmd_args)
+
```

Comment:
Can we test here also that we are getting the 'skipping' log? 
Or maybe worth adding that as a test in the TestDocReviewPrinting class.

Reply:
Addressed in [`7c2c684` (#2660)](https://github.com/demisto/demisto-sdk/pull/2660/commits/7c2c6845dc075daef93be4e0514e29648c02bc8f).